syncd/syncd.cpp

```diff
@@ -1715,6 +1715,7 @@ void handlePortMap(const std::string& portMapFile)
         std::string name, lanes, alias;
         iss >> name >> lanes >> alias;
 
+        iss.clear();
```

Comment:
shall this line after the `iss.str(lanes);`? btw, is it okay to use `tokenize()` function?